assets/css/style.css

```diff
@@ -1,12 +1,12 @@
-.mlBtn {
```

Comment:
JS - Camel case - camelCase
HTML & CSS - Snake case - snake_case

thats the naming convention we're supposed to use

Dan mentions this in the API 2 lecture around 2:08:00